README.md

```diff
@@ -409,6 +409,8 @@ For information about the governance of the Node.js project, see
   **Moshe Atlow** <<moshe@atlow.co.il>> (he/him)
 * [MrJithil](https://github.com/MrJithil) -
   **Jithil P Ponnan** <<jithil@outlook.com>> (he/him)
+* [p-mcgowan](https://github.com/p-mcgowan) -
```

Comment:
Hey @p-mcgowan, thanks for opening this PR, I'll take a look ASAP!

Only one important note: the term collaborator has a specific meaning and is not the same as "contributor".
Please remove this change from the commit 😊
You can find more details about this point here: https://github.com/nodejs/node/blob/main/GOVERNANCE.md#collaborators

Reply:
Well that's embarrassing, thanks - I've removed it.